crates/sui-indexer-alt-jsonrpc/src/args.rs

```diff
@@ -40,7 +40,7 @@ pub enum Command {
         metrics_args: MetricsArgs,
 
         #[command(flatten)]
-        write_args: Option<WriteArgs>,
+        write_args: WriteArgs,
```

Comment:
as `flatten` is to group clap args, it seems to me that we should almost never have `Option` here and should have `Option` if needed on the `Arg` level, what do yall think?

Reply:
It seems like it! But do we know what Option means in the context of flatten in that case?